lib/calc_route.dart

```diff
@@ -0,0 +1,137 @@
+import 'dart:math';
+import 'package:args/args.dart';
+import 'cart_point.dart';
+import 'route_model.dart';
+
+import 'package:sail_routing_dart/route_writer.dart';
+
+bool v_flag = false;
+
+void main(List<String> args) {
+  var parser = new ArgParser();
+  parser.addFlag("verbose", abbr: 'v', defaultsTo: false);
+  var results = parser.parse(args);
+  v_flag = results['verbose'];
+
+  Cart_Point start = new Cart_Point(0.0, 0.0);
+  Cart_Point end = new Cart_Point(1.0, -10.2);
+  double wind = 3 * pi / 2;
+  RouteModel route = new RouteModel(start: start, end: end, wind_radians: wind);
+
+  calculateOptimal(route);
+  print(route);
```

Comment:
Should we have print statements in here?

Reply:
Yes, that is a testing script.